content/contact/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+breadcrumbsImage:
+    title: Contact Us
+    list1: Home
+    list2: contact us
+    image: /assets/img/page-title/page-title-7.jpg
+
+contactArea:
+    span: Contact info
+    title: Have Any Questins Or Emergency Problem Contact With Us
+    description: Pulvinar senectus morbi quisque nunc to towa faucibus netus etiam mone lestie nisi dis malesuada maecenas ora pretium ornare pharetra vestibulum mattis fringilla interdum cursus curae nisi pede laoreet placerat
```

Comment:
no need to give contact area in cms and use it directly in layout when asked we can see that because as hugo.toml file has all the common contact details from there it can be edited

Reply:
i have given description only in cms